### PR TITLE
Airdeploy

### PR DESCRIPTION
**Pull request by thealveen**
@jeronimo13 This is good enough for now. Let's scrap versioning until 3.1 or later.

This still needs your eyes on removing languages that are not going to be published yet (e.g. Swift) from the Flagger SDKs section and the API reference.

### CODE REVIEW

**Comment by jeronimo13** [comment]
Thank for the copy editing!

I updated the flagger part of the doc(I didn't look at anything else)

File: website/static/css/custom.css [hunk 1]
```diff
@@ -4,9 +4,13 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
- 
+
 /* your custom css */
 
+.post a {
+  color: #717171;
```
Comment (jeronimo13) [inline website/static/css/custom.css:11]: @thealveen please, change it to something more suitable. Links look the same as the text in posts which is not great

File: docs/flagger-sdk/default-variation.md [hunk 2]
```diff
@@ -4,20 +4,19 @@ title: Default Variation
...
 }
 ```
 
-## Why do I get default variation?
+The default variation can be returned for a number of reasons, including:
 
-- Your flag's kill switch is on
-- entity/group entity is in the blacklist
-- entity/group entity is not sampled in the population
-- entity filters are not matched with the flag filters
```
Comment (jeronimo13) [inline docs/flagger-sdk/default-variation.md:23]: Am I missing something or this line should be in?

File: docs/api-reference/overview.md [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-
-[TypeScript](ts.md)
-
-[Java](java.md)
-
-[Ruby](ruby.md)
-
-[Python](python.md)
-
-[Golang](go.md)
```
Comment (thealveen) [inline docs/api-reference/overview.md]: @jeronimo13 This cannot be deleted because the top bar references it. Site does not build right now

Comment (jeronimo13) [reply docs/api-reference/overview.md]: I removed header links. It makes sense to have them if we have different general sections of the site like blog, docs, testimonials, etc.

File: docs/api-reference/react.md [hunk 4]
```diff
@@ -54,9 +54,10 @@ if not, make sure to provide a user / entity object to the flag component.
...
 ## FlagSwitch
 
-FlagSwitch component is useful, especially for multivariate flags that are not simply on/off - 
-and have more treatments to handle. 
+FlagSwitch component is useful, especially for multivariate flags that are not simply on/off -
+and have more treatments to handle.
 It's used in combination with the Flag component.
+
 ```javascript
 <FlagProvider envKey="YOUR_ENV_KEY" entity={user}>
```
Comment (thealveen) [inline docs/api-reference/react.md:62]: There are a few instances where this prop is called `envKey` in the documentation, and a few places where it's `apiKey`

File: docs/flagger-sdk/default-variation.md (hunk 2 above)
Comment (thealveen) [reply docs/flagger-sdk/default-variation.md:23]: Not matching filters is an equivalent statement to not being in the subpopulation, which is the condition above

File: docs/api-reference/overview.md (hunk 3 above)
Comment (thealveen) [reply docs/api-reference/overview.md]: Okay

File: docs/api-reference/react.md (hunk 4 above)
Comment (thealveen) [reply docs/api-reference/react.md]: I fixed this

Comment (jeronimo13) [reply docs/api-reference/react.md]: Good eye, it should be apiKey everywhere

File: website/static/css/custom.css (hunk 1 above)
Comment (thealveen) [reply website/static/css/custom.css]: Fixed, thanks for showing me where to override

File: docs/flagger-sdk/default-variation.md (hunk 2 above)
Comment (jeronimo13) [reply docs/flagger-sdk/default-variation.md:23]: Yeah, you are right, closing this

**Pull request merged**